docs_app/src/app/shared/search-results/search-results.component.ts

```diff
@@ -45,14 +71,16 @@ export class SearchResultsComponent implements OnChanges {
     }
```

Comment:
Since you added check in `ngOnChanges`, this part: `if (!search) { return []; }` can be removed.